documentation/List-of-Action-plugins.md

```diff
@@ -246,12 +246,17 @@ When an applicant re-submits an application after making changes, this Action up
 
 - _Action Code:_ **`updateReviewsStatuses`**
 
-| Input parameters<br />(\*required) <br/>                        | Output properties |
-| --------------------------------------------------------------- | ----------------- |
-| `applicationId`                                                 | `updatedReviews`  |
-| `changedApplicationResponses` [Array of applicationResponseIds] |                   |
+| Input parameters<br />(\*required) <br/>               | Output properties |
+| ------------------------------------------------------ | ----------------- |
+| `applicationId`                                        | `updatedReviews`  |
+| `reviewId`                                             |                   |
+| `changedResponses`\* [Array of applicationResponseIds] |                   |
 
-**Note:** If `applicationId` is not provided, the plugin will attempt to fetch it from `applicationData`
+**Note:** - If `applicationId` or `reviewId` is not provided, the plugin will attempt to fetch it from `applicationData`
+**Note 2:** - If `reviewId` is received, the Action will be updating status of related reviews of same stage in the current and next level reviews.
```

Comment:
Hmm good point. The way this is configured, it will run as a Review submission in this case, so maybe we need to prevent that somehow when an application is submitted (and the `reviewId` is also passed through `applicationData`)

I'll update to docs

Reply:
> so maybe we need to prevent that somehow when an application is submitted (and the `reviewId` is also passed through `applicationData`

In which case, should we implement the explicit `isReview` parameter as per some of the other Actions?

Reply:
Have done something else as per [this suggestion](https://github.com/openmsupply/application-manager-server/pull/368#discussion_r636553093) by Andrei: 
> would prefer something a little bit more descriptive, `isTriggeredByReview` or `triggeredBy: 'REVIEW' | 'APPLICATION'`